tools/XmlDocToMarkdown/XmlDocToMarkdown/Program.cs

```diff
@@ -79,11 +97,13 @@ private static void GenerateDocYaml()
             using (var tw = File.CreateText(ymlPath))
             {
                 tw.WriteLine("site_name: Dynamo");
-                tw.WriteLine("pages:");
-                tw.WriteLine("- Home: index.md");
+                tw.WriteLine("repo_url: http://ramramps.github.io/DocumentTesting/");
```

Comment:
These need to be updated for production. And let's move these as strings to the top of the file.